hiss/volunteer/urls.py

```diff
@@ -0,0 +1,13 @@
+from django.urls import path
+
+from volunteer import views
+
+app_name = "volunteer"
+urlpatterns = [
+    path("login", views.EmailObtainAuthToken.as_view(), name="login"),
+    path("checkin", views.CheckinHackerView.as_view(), name="user-checkin"),
+    path("food", views.CreateFoodEventView.as_view(), name="food"),
+    path("workshops", views.CreateWorkshopEventView.as_view(), name="workshops"),
+    path("search", views.SearchView.as_view(), name="search"),
```

Comment:
are these URL's api endponts or front end URL's?

Reply:
They can be both, but in this case they're API endpoints.